kubernetes/schemas/input-allOf

```diff
@@ -0,0 +1,62 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Can you also add a file similar to https://github.com/aavarghese/opa-schema-examples/blob/main/kubernetes/policy-anyOf.rego for `allOf` example that will be described in the schemas OPA documentation?